### ISSUE TITLE
emoji's are appended at the starting position of the textbox

### ISSUE THREAD

**Issue by bhargav794** [suyalamritanshu/Realtime-Chat-App#26]
After writing some text and using emoji's the emojis are restarting from the beginning of the textBox.Look at the video to get a better picture

https://user-images.githubusercontent.com/66371186/136845414-c174bc23-981d-449f-b5ce-c1edc48d1f44.mp4

**Comment by suyalamritanshu** [comment]
Yes you can style them like it is showing in one side you can place it like
whole to one side. Styling you can do yourself as per your experience. My
priorities are project should look good!!

**Comment by suyalamritanshu** [comment]
Yup

**Comment by suyalamritanshu** [comment]
Any updates?

**Comment by bhargav794** [comment]
> Any updates?

Sorry but the problem seems to be unsolvable I've tried a lot of different ways but no use I've also observed that discord has a similar problem in it.
